### PR TITLE
Update enzyme-testing with Testing Automation link.md

### PR DESCRIPTION
**Pull request by abhinashgiri**

### CODE REVIEW

**Comment by abhinashgiri** [comment]
#951 Add links to Test Automation University into Testing Lesson

**Comment by abhinashgiri** [comment]
Updated

File: testing-and-tdd/enzyme-testing.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Here are links to lessons that should be completed before this lesson:
 #### Supplemental Resource
 - [JS Playground's 5 free videos on React testing](https://www.youtube.com/watch?v=aSQ8v9JH5C8&list=PLGDf0elkI13EfDa45q-q1YpAIMBl5mjab)
 - [React Testing with Jest and Enzyme](https://www.udemy.com/react-testing-with-jest-and-enzyme/) donated by the awesome Bonnie Schulkin.
-##### Want to dive deep into learning testing and automation, do check [Test Automation University - Applitools](https://testautomationu.applitools.com/)
+-  Want to dive deep into learning testing and automation, do check [Test Automation University - Applitools](https://testautomationu.applitools.com/)
```
Comment (Kundan28) [inline testing-and-tdd/enzyme-testing.md]: Remove extra white space after the first `-`

**Comment by Kundan28** [comment]
Updated it in scoresheet.

**Comment by alodahl** [comment]
Oh, and be sure to add your issue number or we may make a mistake about points! Can you add it to the description on this PR now even though it has been merged?

**Comment by Kundan28** [comment]
@alodahl I think he already did in the second comment.

**Comment by alodahl** [comment]
yes but it is a lot easier to find in the description

**Comment by Kundan28** [comment]
> Ah, thanks! but it is a lot easier to find in the description. Also, `updated` is not a keyword, so I changed it to `fixes`.

What can't I edit other's comment ?

**Pull request merged**